static/js/tab_bar.js

```diff
@@ -68,7 +68,7 @@ function build_tab_bar(filter) {
         exports.open_search_bar_and_close_narrow_description();
     } else {
         const tab_bar_data = make_tab_data(filter);
-        display_tab_bar(tab_bar_data);
+        append_and_display_tab_bar(tab_bar_data);
```

Comment:
Honestly? This might be a little unnecessary and we can just drop this if reviewers feel the same way.